scripts/keras_benchmarks/gpu_mode.py

```diff
@@ -0,0 +1,26 @@
+""" CNTK gpu config required for running keras models in multi gpu mode."""
+import cntk
+
+def cntk_gpu_mode_config(model, num_samples):
+    # create a CNTK distributed trainer
+    model.model._make_train_function()
+    trainer = model.model.train_function.trainer
+    assert (trainer is not None), "Cannot find a trainer in Keras Model!"
```

Comment:
You don't need to do that here (it's not important in this case) but as a general rule for user-facing code, avoid using `assert` and instead raise `ValueError` with an appropriate message.

Reply:
Also: `assert` applies to everything after it, so no need for parentheses (same comment below)